app/models/word.rb

```diff
@@ -1,2 +1,7 @@
 class Word < ApplicationRecord
+  has_many :lesson_words, dependent: :destroy
+  has_many :lessons, throught: :lesson_words
```

Comment:
not `throught` but
![Screen Shot 2020-03-27 at 10 20 58 PM](https://user-images.githubusercontent.com/2122175/77771469-4ccbcd80-7079-11ea-8794-fa7b542c6731.png)